### ISSUE TITLE
Parse error in Post.php (Timber 1.8.2)

### ISSUE THREAD

**Issue by thijsdejonge** [timber/timber#1850]
### Expected behavior
To run without crashing the entire site.

### Actual behavior
`Parse error: syntax error, unexpected '.', expecting '&' or variable (T_VARIABLE) in public_html/wp-content/plugins/timber-library/lib/Post.php on line 1092`

### Steps to reproduce behavior
Install fresh WP installation
Install timber 1.8.2

### What version of WordPress, PHP and Timber are you using?
WordPress 4.9.8
Timber 1.8.2
PHP 5.4.45

### How did you install Timber? (for example, from GitHub, Composer/Packagist, WP.org?)
Updated to the newest version through WP

**Comment by jarednova** [comment]
Thanks @Toxyc! It would seem something in this function definition is causing an error for you:

`protected function get_revised_data_from_method( $method, ...$args ) {`

Do any PHP experts have ideas on how to make this more version-friendly?

**Comment by Toxyc** [comment]
All I know is that within the last 4 sites I've tried to update they all abruptly killed the website. Not sure how I'm the only one having this issue tho :(

**Comment by jarednova** [comment]
Yeah, definitely want to get to the bottom of this one asap. Here's the offending commit: https://github.com/timber/timber/commit/6703ae4635f917c6068a12c3702cc2315453e882

**Comment by jarednova** [comment]
@Toxyc can you double-check your PHP version? It appears the code should work in PHP 5.6 and greater:

http://php.net/manual/en/migration56.new-features.php

I think we can adapt it for PHP 5.5 and lower for 1.x

**Comment by palmiak** [comment]
As a quickfix wouldn't be enough to change:
```php
protected function get_revised_data_from_method( $method, ...$args ) { 
//to
protected function get_revised_data_from_method( $method, $args ) {
if( !is_array($args) ) {
return;
}
```
as I understand $args should always be an array - and if not we should exit the method because something is wrong.

**Comment by Toxyc** [comment]
My apologies: correct PHP version is this:
**PHP 5.4.45**

I forgot I'm using our old servers to develop, production servers are 7.2 but these are still on 5.4.45 because of old websites running on Kohana.

Is there a way like @palmiak said to fix this for PHP 5.4.45?

If not it's not a big issue I'll just keep working with 1.8.1 in development but it would be convenient to see this fixed.

**Comment by Toxyc** [comment]
For anyone running into this issue: if you are able to upgrade your servers to 7.2 without causing too much trouble I'd definitely recommend that. But as for me I can't fix 300 old Kohana sites from previous developers, so I'm stuck with using 1.8.1 for now.

**Comment by jarednova** [comment]
Fix is in, running tests now:

https://github.com/timber/timber/pull/1851

**Comment by jarednova** [comment]
Released as [Timber 1.8.3](https://github.com/timber/timber/releases/tag/1.8.3)! Thanks again for the report. Timber 2.0 will require PHP 7, but for now, want to maintain the support we've had for 5.5 in 1.x

**Comment by palmiak** [comment]
@Toxyc is the 1.8.3 version workingo for you?
